qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -12,12 +12,13 @@
 
 """The Fermionic-particle Operator."""
 
+import re
 from typing import List, Optional, Tuple, Union
 
 import numpy as np
 
 from qiskit_nature import QiskitNatureError
-from .particle_op import ParticleOp
+from qiskit_nature.operators.second_quantization.particle_op import ParticleOp
```

Comment:
Why did you change the import here?

Reply:
absolute import is better than relative import